RocketMan/zookeeper
# Ericg/57 simplify playlist creation and updating

57: simplify edit & new playlist format

   * single page for playlist edit, delete & restorePlaylist
   * separate GET & POST operations for code clarity
   * normalize form element layout & styling
   * use single quote when possible
   * remove new DJ air name
   * round new show start time to now mod 1/4 hour
   * replace tab in playlist picker with a select element for active & deleted
   * remove tables check code

TODO
   * initialize new show start/end times to that of last week's show (if any)
   * decide whether new DJ Airname should be available in playlist edit screen
     or for clarity done in a different view?

Test Cases:
   * full CRUD on Playlists
   * edit play list elements, eg start time, show name ....
   * confirm add/remove track on a new playlists
   * any legacy operations missing in the new version?
